network-formula/pillar.example

```diff
@@ -32,7 +32,12 @@ network:
         - fe80::1/64
   # each listed route will be written into /etc/sysconfig/network/routes and applied
   routes:
-    default:
+    - destination: default
       gateway: 192.168.120.1
-    10.0.10.10/32:
+    - destination: default
```

Comment:
I understand why you do this change, but it comes with the disadvantage that you have to define the whole routes pillar at once (you can't merge lists from multiple pillar files, while you can easily do this for dicts).

Just as a crazy idea: what about keeping the dict style and allowing `default4:` and `default6:` which would then both be written out as `default` routes?
(Of course this only makes sense if the default route is the only route that needs duplicate keys, but so far I haven't seen other cases that would need duplicate keys.)

If you think this idea is too crazy, feel free to merge as is.

Reply:
Really, not with `pillar_merge_lists`?

https://docs.saltproject.io/en/latest/ref/configuration/master.html#pillar-merge-lists

Yes, `default4` and `default6` would have been my backup plan. There shouldn't be any other routes that could exist as duplicates.

Reply:
Maybe the extra logic is not too bad after all, what do you think?

https://github.com/tacerus/opensuse-salt-formulas/commit/b0999564c6cffbe229dbdab6cb11db249eccf3de

Reply:
Updated this branch.